rules/linux/builtin/lnx_nimbuspwn_privilege_escalation_exploit.yml

```diff
@@ -0,0 +1,23 @@
+title: Nimbuspwn Exploitation
+id: 7ba05b43-adad-4c02-b5e9-c8c35cdf9fa8
+status: experimental
+description: Detects exploitation of Nimbuspwn privilege escalation vulnerability (CVE-2022-29799 and CVE-2022-29800)
+author: Bhabesh Raj
+references:
+  - https://www.microsoft.com/security/blog/2022/04/26/microsoft-finds-new-elevation-of-privilege-linux-vulnerability-nimbuspwn/
+  - https://github.com/Immersive-Labs-Sec/nimbuspwn
+date: 2022/05/04
+logsource:
+  product: linux
+detection:
+  keywords|contains|all:
```

Comment:
Hi,
here `keywords` is for a [list](https://github.com/SigmaHQ/sigma/wiki/Specification#lists).
You can use `Value Modifiers` only on [fields](https://github.com/SigmaHQ/sigma/wiki/Specification#value-modifiers)
convert give :
![image](https://user-images.githubusercontent.com/62423083/166863751-d0b464c6-c179-4bc2-b680-2bf4cb93ff46.png)

without modifiers:
![image](https://user-images.githubusercontent.com/62423083/166863824-33c92dc3-8106-47f4-95f1-70a0ad857049.png)